### PR TITLE
Make compatible with `curated_cof_version=2` 

### PR DESCRIPTION
**Pull request by danieleongari**

### CODE REVIEW

File: select-figure/main.ipynb [hunk 1]
```diff
@@ -55,44 +55,41 @@
...
-    "    uuids = get_data_aiida()\n",
-    "    uuid_dict = { v[0]: v[1:] for v in uuids }\n",
-    "\n",
-    "    # label, ref_uuid, opt_uuid, pe_uuid\n",
-    "    uuids_pe = get_data_aiida(has_pe=True)\n",
-    "    uuid_dict_pe = { v[0]: v[1:] for v in uuids_pe }\n",
-    "\n",
+    "    # import cof-frameworks as DataFrame\n",
+    "    url = 'https://raw.githubusercontent.com/danieleongari/CURATED-COFs/additions/cof-frameworks.csv' #TODO: update\n",
+    "    df = pandas.read_csv(url) # DEPRECATED: pandas.read_csv(this_dir + \"static/cof-frameworks.csv\")\n",
```
Comment (ltalirz) [inline select-figure/main.ipynb]: this is something I wouldn't recommend.
not only can this slow down things, it also means existing apps can break whenever we decide to change something about the format here.

once we move to a pipeline for updating things on the fly, we can revisit these things, but for the moment let's keep a local copy

**Review by ltalirz — CHANGES_REQUESTED** [review]
thanks @danieleongari , I've had a look through and just one main comment.

We may need to look into timings of queries at some point but this we can do when we move this to production.

**Comment by danieleongari** [comment]
Ok, I will copy new csv in static again (also cof-papers.csv).

**Comment by ltalirz** [comment]
thanks & merged!

**Pull request merged**